torch/csrc/jit/passes/quantization.cpp

```diff
@@ -544,9 +544,9 @@ std::tuple<IValue, IValue> QuantizeHelper::getQParams(Value* v) {
       v->debugName(),
       " exists.");
   auto om = module_.attr(observer_name.value()).toModule();
-  auto calculate_qparams = om.get_method("calculate_qparams");
-  IValue qparams = calculate_qparams(std::vector<IValue>());
-  checkCalculateQParamsResult(qparams);
+  auto get_qparams = om.get_method("get_qparams");
```

Comment:
NIT: Might be better to use a more descriptive name than 'om', perhaps: 'observer_module'